src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-overview-dashboard/rgw-overview-dashboard.component.html

```diff
@@ -0,0 +1,80 @@
+<div class="container-fluid">
+  <div class="row">
```

Comment:
first column starts here
```suggestion
  <div class="row">
    <div class="col-lg-3">
```

Reply:
Done